pages.fa/common/chmod.md

```diff
@@ -0,0 +1,36 @@
+# chmod
+
+> تغییر مجوز(ها)ی دسترسی به یک فایل یا پوشه
+> اطلاعات بیشتر : <https://www.gnu.org/software/coreutils/chmod>.
+
+- به مالک فایل دسترسی اجرا میدهد
+
+`chmod u+x {{مسیر/فایل}}`
+
+- به کابر مالک دسترسی خواند|نوشتن یک فایل|پوشه را میدهد
+
+`chmod u+rw {{مسیر/فایل/یا/پوشه}}`
+
+- دسترسی اجرا را از گروه صلب میکند
+
+`chmod g-x {{مسیر/فایل}}`
+
+- به تمامی کاربرها دسترسی خواندن و اجرا میدهد
+
+`chmod a+rx {{مسیر/فایل}}`
+
+- به دیگران(کاربرانی که صاحب فایل نیستند) دسترسی های گروه را میدهد
+
+`chmod o=g {{مسیر/فایل}}`
+
+- به همگان همه دسترسی(ها) را میدهد
+
+`chmod o= {{مسیر/فایل}}`
+
+- به صورت بازگشتی به گروه و دیگران دسترسی نوشتن میدهد
+
+`chmod -R g+w,o+w {{مسیر/فایل}}`
```

Comment:
Shouldn't it be `{{path/to/directory}}`'s native equivalent here instead of `{{path/to/file}}`?

Edit from chown file.

```suggestion
`chmod -R g+w,o+w {{مسیر/به/پوشه}}`
```

Reply:
Will review and correct.